idd/Energy+.idd.in

```diff
@@ -19197,6 +19200,408 @@ RoomAirSettings:UnderFloorAirDistributionExterior,
+      \key ElectricLoadCenter:Storage:Simple
+      \key Pipe:Indoor
+      \key Refrigeration:Case
+      \key Refrigeration:CompressorRack
+      \key Refrigeration:System:Condenser:AirCooled
+      \key Refrigeration:TranscriticalSystem:GasCooler:AirCooled
+      \key Refrigeration:System:SuctionPipe
+      \key Refrigeration:TranscriticalSystem:SuctionPipeMT
+      \key Refrigeration:TranscriticalSystem:SuctionPipeLT
+      \key Refrigeration:SecondarySystem:Receiver
+      \key Refrigeration:SecondarySystem:Pipe
+      \key Refrigeration:WalkIn
+      \key Pump:VariableSpeed
+      \key Pump:ConstantSpeed
+      \key Pump:VariableSpeed:Condensate
+      \key HeaderedPumps:VariableSpeed
+      \key HeaderedPumps:ConstantSpeed
+      \key ZoneContaminantSourceAndSink:GenericContaminant
+      \key PlantComponent:UserDefined
+      \key Coil:UserDefined
+      \key ZoneHVAC:ForcedAir:UserDefined
+      \key AirTerminal:SingleDuct:UserDefined  
+  A5, \field Internal Gain Object 2 Name
+      \type alpha
+  N2, \field Fraction of Gains to Node 2
+      \type real
+      \mininum 0.0
+      \maximum 1.0
+  A6, \field Internal Gain Object 3 Type
+      \type choice
+      \key People
+      \key Lights
+      \key ElectricEquipment
+      \key GasEquipment
+      \key HotWaterEquipment
+      \key SteamEquipment
+      \key OtherEquipment
+      \key ZoneBaseboard:OutdoorTemperatureControlled
+      \key ZoneContaminantSourceAndSink:CarbonDioxide
+      \key WaterUse:Equipment     
+      \key DaylightingDevice:Tubular
+      \key WaterHeater:Mixed
+      \key WaterHeater:Stratified
+      \key ThermalStorage:ChilledWater:Mixed
+      \key ThermalStorage:ChilledWater:Stratified
+      \key Generator:FuelCell
+      \key Generator:MicroCHP
+      \key ElectricLoadCenter:Transformer
+      \key ElectricLoadCenter:Inverter:Simple
+      \key ElectricLoadCenter:Inverter:FunctionOfPower
+      \key ElectricLoadCenter:Inverter:LookUpTable
+      \key ElectricLoadCenter:Storage:Battery
+      \key ElectricLoadCenter:Storage:Simple
+      \key Pipe:Indoor
+      \key Refrigeration:Case
+      \key Refrigeration:CompressorRack
+      \key Refrigeration:System:Condenser:AirCooled
+      \key Refrigeration:TranscriticalSystem:GasCooler:AirCooled
+      \key Refrigeration:System:SuctionPipe
+      \key Refrigeration:TranscriticalSystem:SuctionPipeMT
+      \key Refrigeration:TranscriticalSystem:SuctionPipeLT
+      \key Refrigeration:SecondarySystem:Receiver
+      \key Refrigeration:SecondarySystem:Pipe
+      \key Refrigeration:WalkIn
+      \key Pump:VariableSpeed
+      \key Pump:ConstantSpeed
+      \key Pump:VariableSpeed:Condensate
+      \key HeaderedPumps:VariableSpeed
+      \key HeaderedPumps:ConstantSpeed
+      \key ZoneContaminantSourceAndSink:GenericContaminant
+      \key PlantComponent:UserDefined
+      \key Coil:UserDefined
+      \key ZoneHVAC:ForcedAir:UserDefined
+      \key AirTerminal:SingleDuct:UserDefined  
```

Comment:
@mjwitte The UserDefined components should not be used in both places. Otherwise, it will cause double counting. Since the components were listed before I took over, I just uses as is.